lib/modules/dosomething/dosomething_kudos/includes/Kudos.php

```diff
@@ -86,6 +93,16 @@ public static function find(array $filters = []) {
   }
 
   /**
+   * Get the approved reactions.
+   * @return array
+   */
+  public static function getApprovedReactions() {
+    $kudos = new static;
```

Comment:
Why not make `$approved_reactions` static so you don't have to make and discard an instance here?